### ISSUE TITLE
Request timeout for customs scanner

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-server#15600]
We're seeing some 504s lately. I am not sure why but maybe we should implement a retry strategy like we did for MAD.

---

https://sentry.prod.mozaws.net/operations/olympia-prod/issues/9351575/?referrer=github_plugin

```
ValueError: {'message': 'Endpoint request timed out'}
  File "olympia/scanners/tasks.py", line 70, in run_scanner
    _run_scanner_for_url(
  File "olympia/scanners/tasks.py", line 119, in _run_scanner_for_url
    raise ValueError(data)

Error in scanner "customs" task for FileUpload 3710116.
```

**Comment by willdurand** [comment]
@diox what do you think about adding a retry strategy?

**Comment by diox** [comment]
I'd prefer if we didn't need that, but we could use a retry strategy similar to what we do for mad, as you suggested in the standup. It'll be interesting to see if that solves anything.

**Comment by willdurand** [comment]
> I'd prefer if we didn't need that, but we could use a retry strategy similar to what we do for mad, as you suggested in the standup. It'll be interesting to see if that solves anything.

I am not sure I fully understand your comment but I agree we should add the retry strategy for 50x like we do for MAD (and I thought that it was what I suggested)

**Comment by AlexandraMoga** [comment]
@willdurand I've submitted several add-ons while focusing mostly on large `.js` files. I've used [this add-on example](https://code.addons-dev.allizom.org/en-US/browse/620206/versions/1692142/) and made parallel submissions on -dev and stage. On stage, I received the "Sorry we could not load your webextension" message almost constantly while on -dev the upload was successful. 
This looks like an improvement to me, but let me know if you know of other use cases that used to cause the upload error, besides large `.js` files.
